tests/unit_tests/test_server_client.py

```diff
@@ -1960,10 +1970,10 @@ async def test_endpoint_configurations() -> None:
         ("POST", "/c/1234/batch", {"inputs": [1, 2]}),
         ("POST", "/c/1234/stream", {"input": 1}),
         ("POST", "/c/1234/stream_log", {"input": 1}),
-        ("POST", "/c/1234/input_schema", {}),
-        ("POST", "/c/1234/output_schema", {}),
-        ("POST", "/c/1234/config_schema", {}),
-        ("POST", "/c/1234/playground/index.html", {}),
+        ("GET", "/c/1234/input_schema", {}),
```

Comment:
why did this change?

Reply:
This was incorrect before. I need to check why the test before didn't catch it

Reply:
OK fixed the test to be more specific